src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/References/UnSetAttributeCommand.cs

```diff
@@ -1,50 +1,16 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
-using System.Linq;
-using System.Threading.Tasks;
 using Microsoft.VisualStudio.ProjectSystem.Properties;
 
 namespace Microsoft.VisualStudio.ProjectSystem.VS.References
 {
-    internal class UnSetAttributeCommand : IReferenceCommand
+    internal class UnsetAttributeCommand : BaseSetAttributeCommand
```

Comment:
The file name's capitalization should also be updated.

Reply:
If I change the name from Un**S**etAttributeCommand.cs to Un**s**etAttributeCommand.cs  Git doesn't detect this  change.